wolfcrypt/src/port/nxp/se050_port.c

```diff
@@ -501,6 +501,72 @@ static sss_algorithm_t se050_map_hash_alg(int hashLen)
     return algorithm;
 }
 
+int se050_ecc_insert_private_key(int keyId, const byte* eccDer,
```

Comment:
Consider using `se050_allocate_key(SE050_ECC_KEY` and returning the assigned `keyId`?

Reply:
> Consider using `se050_allocate_key(SE050_ECC_KEY` and returning the assigned `keyId`?

This wouldn't work. Because the keyId returned would start at 100. So our private key would be inserted permanently at ID 100. On the next boot the first temporary key will also try to use key ID 100, it will silently fail and then later error. This is why I'm trying to force permanent storage to use IDs less than 100.

Now, if we have another type such as `SE050_ECC_PRIVATE_KEY` that uses say 0-50 as IDs, that could work. I'm not sure if there is an advantage to this over a user-set one for the usage of this function though (I can be convinced otherwise).